{% extends 'catalog/base.html' %}
{% load static %}

{% block title %}Polygenic Score Data Description{% endblock %}

{% block content %}
    <nav aria-label="You are here:">
      <ol class="breadcrumb">
        <li class="breadcrumb-item"><a href="../../">PGS Catalog</a></li>
        <li class="breadcrumb-item">Documentation</li>
        <li class="breadcrumb-item">Data Description</li>
      </ol>
    </nav>

    <h2>PGS Catalog Data Description</h2>
    <p>
      This page contains information regarding the contents of the PGS Catalog.
    </p>
    <div class="content_container">
      <div class="table-of-contents">
        <nav role="navigation">
          <h5>Table of Contents</h5>
          <ul>
            <li><a href="#desc_pub">Publication</a></li>
            <li><a href="#desc_pgs">Polygenic Score (PGS)</a></li>
            <li><a href="#desc_trait">Trait</a></li>
            <li><a href="#desc_smpl">Sample</a></li>
            <li><a href="#desc_perf">Performance Metrics</a></li>
          </ul>
        </nav>
      </div>
    </div>


    <section>
        <br />
        <p>In this section we describe the curated data and tables extracted from the PGS publications and presented in the catalog.</p>

        <h4 id="desc_pub">Publication</h4>
        <p>Each publication in the database is given a <u><b>PGP</b> (Polygenic Publication ID)</u> so that scores and evaluations link to the same source object.
        When browsing by publications the <u>Number of PGS Developed</u> refers to the number of newly developed PGS in the paper, and the <u>Number of PGS Evaluated</u>
        refers to the number of PGS (new and existing) that have performance metrics derived in the study. For each publication the following information is extracted:</p>
        <table class="table table-bordered table-left-col-filled">
          <tbody>
            <tr>
              <td><b>PubMed ID</b></td>
              <td>PubMed identification number</td>
            </tr>
            <tr>
              <td><b>doi</b></td>
              <td>A Digital Object Identifier (doi) is also curated to allow unpublished work (e.g. pre-prints) to be added to the catalogue</td>
            </tr>
            <tr>
              <td><b>Publication Title</b></td>
              <td>Title of the publication</td>
            </tr>
            <tr>
              <td><b>Author(s)</b></td>
              <td>List of publication authors, the first author is also extracted for a shorter display</td>
            </tr>
            <tr>
              <td><b>Journal</b></td>
              <td>The publication's location</td>
            </tr>
            <tr>
              <td><b>Publication Date</b></td>
              <td>Date of publication (with respect to the PMID or doi upon DB upload)</td>
            </tr>
          </tbody>
        </table>

        <br />
        <h4 id="desc_pgs">Polygenic Score (PGS)</h4>
        <p>Each publication in the database is given a unique <u><b>PGS</b> (Polygenic Score) ID</u> to identify it. The following information is extracted, and associated with each PGS in the catalog:</p>

        <table class="table table-bordered table-left-col-filled">
          <tbody>
            <tr>
              <td><b>Source (PGP ID)</b></td>
              <td>A PGP ID links the PGS to the publication in which it was described.</td>
            </tr>
            <tr>
              <td><b>Reported Trait</b></td>
              <td>The author-reported trait (e.g. body mass index (BMI), or coronary artery disease) that the PGS has been developed to predict.</td>
            </tr>
            <tr>
              <td><b>Mapped Trait(s) / EFO ID(s)</b></td>
              <td>The <u>Reported Trait</u> is mapped to <a href="https://www.ebi.ac.uk/ols/ontologies/efo">Experimental Factor Ontology (EFO)</a> terms and their respective identifiers. For more information see the <a href="#desc_trait">trait information</a>.</td>
            </tr>
            <tr>
              <td><b>Score Name</b></td>
              <td>This may be the name that the authors describe the PGS with in the source publication, or a name that a curator has assigned to identify the score during the curation process (before a PGS ID has been given).</td>
            </tr>
            <tr>
              <td><b>PGS Development Method</b></td>
              <td>The name or description of the method or computational algorithm used to develop the PGS.</td>
            </tr>
            <tr>
              <td><b>PGS Development Details/Relevant Parameters</b></td>
              <td>A description of the relevant inputs and parameters relevant to the PGS development method/process.</td>
            </tr>
            <tr>
              <td><b>Original Genome Build</b></td>
              <td>The version of the genome that the variants present in the PGS are associated with. Listed as NR (Not Reported) if unknown.</td>
            </tr>
            <tr>
              <td><b>Number of Variants</b></td>
              <td>Number of variants used to calculate the PGS. In the future this will include a more detailed description of the types of variants present.</td>
            </tr>
            <tr>
              <td><b>Number of Variant Interaction Terms</b></td>
              <td>Number of higher-order variant interactions included in the PGS.</td>
            </tr>
          </tbody>
        </table>

        <br />
        <p>The following information about the PGS are captured in tables and described in subsequent sections:</p>

        <table class="table table-bordered table-left-col-filled">
          <tbody>
            <tr>
              <td><b>Contributing Samples:</b></td>
              <td>
                <p>Information about the samples used for the development of the PGS. Relevant column descriptions are in the <a href="#desc_smpl">Sample Description</a> section.</p>
                <table class="table table-bordered table-left-col-filled">
                  <tbody>
                    <tr>
                      <td><b>Source of Variant Associations</b> (GWAS)</td>
                      <td>A table describing the samples used to define the variant associations/effect-sizes used in the PGS. These data are extracted from the GWAS Catalog when a study ID (GCST) is available.</td>
                    </tr>
                    <tr>
                      <td><b>Score Development/Training</b></td>
                      <td>A table describing the samples used to develop or train the score (e.g. not used for variant discovery, and non-overlapping with the samples used to evaluate the PGS predictive ability).</td>
                    </tr>
                  </tbody>
                </table>
            </tr>
            <tr>
              <td><b>Performance Metrics</b></td>
              <td>A record of the performance metrics that have been reported for the PGS. Relevant column descriptions are in the <a href="#desc_perf">Performance Metrics</a> section</td>
            </tr>
            <tr>
              <td><b>Evaluated Samples</b></td>
              <td>Information about the samples used in PGS performance evaluation. These samples have an <u><b>PSS</b> (PGS Catalog Sample Set ID)</u> to link them to their associated performance metrics (and across different PGS). Relevant column descriptions are in the <a href="#desc_smpl">Sample Description</a> section.</td>
            </tr>
          </tbody>
        </table>

        <br />
        <h4 id="desc_trait">Trait</h4>
        <p>Traits in the catalog are displayed/grouped according to the <u>Mapped Traits</u> rather than the author <u>Reported Traits</u> to facilitate comparability, similar to the GWAS Catalog.
        For a complete description of why the trait ontology is employed please refer to <a href="https://www.ebi.ac.uk/gwas/docs/ontology">related documentation</a> from the GWAS Catalog.
        The Experimental Factor Ontology is hosted and described here: <a href="https://www.ebi.ac.uk/efo/">Experimental Factor Ontology (EFO)</a>. The information for each EFO trait ID that is stored in the PGS catalog is:</p>

        <table class="table table-bordered table-left-col-filled">
          <tbody>
            <tr>
              <td><b>Experimental Factor Ontology ID</b> (EFO_ID)</td>
              <td>A trait identifier that links to the EFO and GWAS catalog.</td>
            </tr>
            <tr>
              <td><b>EFO Label</b></td>
              <td>The trait label/name</td>
            </tr>
            <tr>
              <td><b>Trait Description</b></td>
              <td>Detailed description of the trait from EFO.</td>
            </tr>
          </tbody>
        </table>

        <br />
        <h4 id="desc_smpl">Sample</h4>
        <p>A consistent set of fields are used to describe the samples used to develop and test each PGS:</p>

        <table class="table table-bordered table-left-col-filled">
          <tbody>
            <tr>
              <td><b>PGS Catalog Sample Set ID</b> (PSS ID)</td>
              <td>PSS IDs are assigned to describe samples used in PGS evaluations (e.g. Performance Metrics). PSS IDs are not uniquely associated with a single PGS, multiple PGS can be evaluated on the sample sample sets (PSS ID).</td>
            </tr>
            <tr>
              <td><b>Study Identifiers</b></td>
              <td>Identifiers used to link the samples to their initial descriptions (e.g. using PubMed IDs) and if they were used for variant associations to their associated GWAS studies (using GWAS Catalog GCST IDs).</td>
            </tr>
            <tr>
              <td><b>Sample Numbers</b></td>
              <td>This field describes the number of individuals included in the sample, along with the number of cases and controls (if the trait is dichotomous) and the percent of participants that are male if available.</td>
            </tr>
            <tr>
              <td><b>Ancestry fields:</b></td>
              <td><p>Fields describing sample ancestry are curated according to the <a href="https://www.ebi.ac.uk/gwas/docs/ancestry">framework</a> used to record ancestry data in the GWAS Catalog that was defined in <a href="https://doi.org/10.1186/s13059-018-1396-2">Morales <i>et al. Genome Biology</i> (2018)</a>:</p>
                <table class="table table-bordered table-left-col-filled">
                  <tbody>
                    <tr>
                      <td><b>Broad Ancestry Category</b></td>
                      <td>Author reported ancestry is mapped to the best matching ancestry category from the GWAS Catalog framework (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5815218/table/Tab1/?report=objectonly">Table 1, Morales <i>et al.</i> (2018)</a>).</td>
                    </tr>
                    <tr>
                      <td><b>Ancestry</b></td>
                      <td>A more detailed description of sample ancestry that usually matches the most specific description described by the authors (e.g. French, Chinese).</td>
                    </tr>
                    <tr>
                      <td><b>Sample Ancestry</b></td>
                      <td>Is displayed on the website, and represents a combination of the two ancestry categories (with the more specific terms in brackets).</td>
                    </tr>
                    <tr>
                      <td><b>Country of recruitment</b></td>
                      <td>Author reported countries of recruitment (if available).</td>
                    </tr>
                    <tr>
                      <td><b>Additional Ancestry Description</b></td>
                      <td>Any additional description not captured in the structured data (e.g. founder or genetically isolated populations, or further description of admixed samples).</td>
                    </tr>
                  </tbody>
                </table>
              </td>
            </tr>
            <tr>
              <td><b>Cohort(s)</b></td>
              <td>A list of cohorts that collected the samples. The initial list of common cohorts used in genetics studies that seeded these annotations is from <a href="https://doi.org/10.1038/s42003-018-0261-x">Mills & Rahal. Communications Biology. (2019)</a>.</td>
            </tr>
            <tr>
              <td><b>Additional Sample/Cohort Information</b></td>
              <td>Any additional description about the samples and what they were used for that is not captured by the structured categories (e.g. sub-cohort information).</td>
            </tr>
          </tbody>
        </table>

        <br />
        <h4 id="desc_perf">Performance Metrics</h4>
        <p>Each evaluation of a PGS is given a <u><b>PPM</b> (PGS Performance Metric) ID</u> that links it to a description of the results:</p>
        <table class="table table-bordered table-left-col-filled">
          <tbody>
            <tr>
              <td><b>PGS Catalog Sample Set (PSS) ID</b></td>
              <td>ID that links to the samples the displayed PGS evaluated.</td>
            </tr>
            <tr>
              <td><b>Source/Publication</b></td>
              <td>ID that links to the publication where the performance metrics were reported. </td>
            </tr>
            <tr>
              <td><b>Trait</b></td>
              <td>This field displays both the <u>Reported</u> and <u>Mapped Traits</u>. The reported trait often corresponds to the test set names reported in the publication, or more specific aspects of the phenotype being tested (e.g. if the disease cases are incident vs. recurrent events).</td>
            </tr>
            <tr>
              <td><b>Reported values:</b></td>
              <td>
              	<p>The reported values of the performance metrics are all reported similarly (e.g. the estimate is recorded along with the 95% confidence interval (if supplied)) and grouped according to the type of statistic they represent:</p>
                <table class="table table-bordered table-left-col-filled">
                  <tbody>
                    <tr>
                		  <td><b>PGS Effect Sizes (per SD change)</b></td>
                		  <td>Examples include regression coefficients (betas) for continuous traits, Odds ratios (OR) and/or Hazard ratios (HR) for dichotomous traits depending on the availability of time-to-event data.</td>
                		</tr>
                		<tr>
                		  <td><b>PGS Classification Metrics</b></td>
                		  <td>Examples include the Area under the Receiver Operating Characteristic (AUROC) or Harrell's C-index (Concordance statistic).</td>
                		</tr>
                		<tr>
                		  <td><b>Other Metrics</b></td>
                		  <td>Metrics that do not fit into the other two categories. Examples include: <i>R<sup>2</sup></i> (proportion of the variance explained), or reclassification metrics.</td>
                		</tr>
              	  </tbody>
                </table>
              </td>
            </tr>
               <tr>
              <td><b>Covariates Included in PGS Model</b></td>
              <td>A comma-separated list of covariates used in the prediction model used to evaluate the PGS. Examples include: age, sex, smoking habits, etc.</td>
            </tr>
            <tr>
              <td><b>PGS Performance: Other Relevant Information</b></td>
              <td>Any other information relevant to the understanding of the performance metrics.</td>
            </tr>
          </tbody>
        </table>

    </section>
{% endblock %}
